velox/functions/prestosql/window/tests/WindowTestBase.cpp

```diff
@@ -98,7 +110,8 @@ void WindowTestBase::assertWindowFunctionError(
 
 void WindowTestBase::testTwoColumnOverClauses(
     const std::vector<RowVectorPtr>& input,
-    const std::string& windowFunction) {
+    const std::string& windowFunction,
+    const std::string frameClause) {
```

Comment:
use std::optional for frameClause.

Reply:
Thanks for the suggestion, replaced it with a bool testFrames and using optional parameters in testWindowFunction.